tests/alembic_migrations/versions/v_efde19ea4f52_fix_function_getstationvariabletable/conftest.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+from sqlalchemy.orm import sessionmaker
+from ....alembicverify_util import prepare_schema_from_migrations
+from ....helpers import insert_crmp_data
+
+
+@pytest.fixture(scope="function")
+def prepared_schema_from_migrations_left(uri_left, alembic_config_left, db_setup):
+    engine, script = prepare_schema_from_migrations(
+        uri_left,
+        alembic_config_left,
+        db_setup=db_setup,
+        revision="efde19ea4f52",
+    )
+
+    yield engine, script
+
+    engine.dispose()
+
+
+@pytest.fixture(scope="function")
```

Comment:
It doesn't look like this gets used.

Reply:
No, it doesn't and worse this pattern is all over the place in the migration smoke tests. I'll put this off to a different issue #201 .